### DISCUSSION TITLE
Cannot update on MacOS Monterey.

### DISCUSSION THREAD

**Discussion by githubtefo** [debauchee/barrier#1475] [Q&A] [answered]
Hi everyone,

I am trying to update to the last version of Barrier, although I can't overwrite the previous instance of barrier on the Applications folder. It keeps saying I have to close the App first, even when I cannot see the process running. I cannot remove the old version neither.

Any thoughts?
Thanks!

**Comment by Malachai7** [comment]
When you close the app are using the keyboard function and at the very top of the bar are you closing it there too? 
if that doesn't work bring up the task manager and kill the process. See if that works for you.

**Comment by githubtefo** [comment, marked as answer]
Finaly was able to update after another restart of the system without opening barrier.
Thanks!
